### DISCUSSION TITLE
What are the best places for discussions ?

### DISCUSSION THREAD

**Discussion by devingfx** [Comfy-Org/ComfyUI#4738] [General]
Hi!

With the recent creation of Comfy-Org, and it's repo mirrors/splits, it's a bit confusing where are the best place to discuss stuff...

Could you, please, tell us what are the most recent best places (aka Github repos) to discuss or open issues concerning:

- Backend => [comfyanonymous/ComfyUI](https://github.com/comfyanonymous/ComfyUI/discussions)
  - Core exection / install / NN stuff => ???
  - API server => [comfyanonymous/ComfyUI](https://github.com/comfyanonymous/ComfyUI/discussions)
  - core Custom nodes API => [Comfy-Org/???](https://github.com/orgs/Comfy-Org/repositories?type=all) ? [comfyanonymous/ComfyUI](https://github.com/comfyanonymous/ComfyUI/discussions) ?
  - Individual Custom Node => each Custom Node repository
  - File formats => ???
  - Tools => [Comfy-Org/???](https://github.com/orgs/Comfy-Org/repositories?type=all)
- Front-end
  - Legacy => [comfyanonymous/ComfyUI](https://github.com/comfyanonymous/ComfyUI/discussions) ? [Comfy-Org/ComfyUI_legacy_frontend](https://github.com/Comfy-Org/ComfyUI_legacy_frontend) ?
  - Core => [Comfy-Org/ComfyUI_frontend](https://github.com/Comfy-Org/ComfyUI_frontend/discussions)
  - API client  => [Comfy-Org/ComfyUI_frontend](https://github.com/Comfy-Org/ComfyUI_frontend/discussions) ?
  - Tutorials => [Comfy-Org/docs](https://github.com/Comfy-Org/docs) ?

_( feel free to add/remove categories from this 1st shot list :) of course )_

**Comment by ltdrdata** [comment]
The frontend issue should be posted following repo:
https://github.com/Comfy-Org/ComfyUI_frontend

Issues related to custom nodes should be addressed in each custom node repository.
(Issues occurring in custom nodes cannot be handled by the ComfyUI core. It is technically impossible and beyond our capacity to handle issues for thousands of custom nodes here.)

This repo is for reporting issues related to the code that is maintained in this repo.
Requests for improvements or discussions other than issues (except frontend thing) can be posted in this repo.

**Reply by devingfx** [reply]
Thx for this quick reply! :)

So you confirm that mirror repos of backend in Comfy-Org (like [ComfyUI-Mirror](https://github.com/Comfy-Org/ComfyUI-Mirror) or [ComfyUI](https://github.com/Comfy-Org/ComfyUI)) are not the places to be ?

> Issues related to custom nodes should be addressed in each custom node repository.

About Custom Nodes, I meant more about core Custom Nodes API, registry etc, I suppose this is to gather in [Comfy-PR](https://github.com/Comfy-Org/Comfy-PR)... (any other?)

> This repo is for reporting issues related to the code that is maintained in this repo.

Yes, thx, I know how git / github / OSS dev works :) the question was more about what's the new organisation of repos between here and [Comfy-Org](https://github.com/orgs/Comfy-Org/repositories?type=all), what's your plans for future, would all repos be centralised in the Comfy-Org or not, etc...

Just to clarify, I started this discussion to get some insights, maybe also permit non-definitive plans to be discussed here, and finally come to an exhaustive list suject=>repo to cook some usefull guideline/docs for contributors...

**Reply by mcmonkey4eva** [reply]
-Mirror is for testing, and the other one... i'm not sure why that's there tbh
Comfy-PR is a project that automatically submits PRs to comfy nodes to add them to the registry

**Comment by devingfx** [comment]
I'll edit the OP each time an advice/answer is given time to time...
